example/java/TestGenomicsDBImporterWithMergedVCFHeader.java

```diff
@@ -32,17 +32,27 @@
 
   public static void main(String[] args) throws IOException, GenomicsDBException, ParseException, InterruptedException {
     // this expects --coalesce-multiple-contigs to be the last argument, if it exists
-    int numPartitions = 0;
+    // similarly --consolidate-only. and they should be mututally exclusive
+    int numPartitions = 0, doConsolidate = -1;
     if (args[args.length-2].equals("--coalesce-multiple-contigs")) {
       numPartitions = Integer.parseInt(args[args.length-1]);
       args = Arrays.copyOf(args, args.length-2);
     }
+    else if (args[args.length-2].equals("--consolidate-only")) {
+      doConsolidate = Integer.parseInt(args[args.length-1]);
+      args = Arrays.copyOf(args, args.length-2);
+    }
```

Comment:
Perhaps move these statements to inside the CommandLineImportConfig class and avoid position dependent if statements?

Reply:
Done. Being lazy there would only lead to annoying test failures later...